### DISCUSSION TITLE
MG not talking to HWInfo

### DISCUSSION THREAD

**Discussion by Joker41NAM** [raiguard/ModernGadgets#258] [Q&A] [answered]
Hello.  Since I updated to 1.8.1, it appears that it's not talking to HWinfo (v6.42) correctly.  All temp and voltage reading for CPU & GPU show as 0, and if I go to "Open HWiNFO sensors list" it says "ERROR:  HWiNFO is not configured or is not running."  But it is running, so I'm not sure where to look.  I've gone through the Configuration page, and everything matches the recommended settings.

Any help would be appreciated.

**Comment by raiguard** [comment, marked as answer]
1.8 requires HWiNFO 7.03 or newer. You will need to reconfigure all of your HWiNFO settings as well.

**Reply by Joker41NAM** [reply]
Ah, I see.  I misunderstood the patch notes; I thought they meant it supported v7, not required it.

**Reply by raiguard** [reply]
I probably could have been more clear about that. Thanks for using ModernGadgets!
